css/index.css

```diff
@@ -143,4 +143,62 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+nav {
+    display: flex;
+    justify-content: space-evenly;  
+    text-decoration: none;
+    margin-left: 40px;
+    margin-right: 40px;
+    align-items: flex-end;
+}
+ header nav a.logo{
+    padding: 1px;
+     width: 70%;
+     height: 90%;
+     padding: 10%;
+ }
+ .my-top-img{
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     width: 70%;
+     padding-top: 5px;
+     padding-bottom: 5px;
+ }
```

Comment:
looks like you have .my-top-img twice, remember that CSS is cascading meaning if you over-write your styles, you will be essentially over-writing previous styles.